### PR TITLE
[CI:DOCS]First pass at review comments

### PR DESCRIPTION
**Pull request by baude**
Tackling the first comments in the review pass.  More to come.

Signed-off-by: baude <bbaude@redhat.com>
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/api/handlers/libpod/pods.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package libpod
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/containers/libpod/pkg/util"
```
Comment (rhatdan) [inline pkg/api/handlers/libpod/pods.go]: Wrong place

File: pkg/api/handlers/utils/errors.go [hunk 2]
```diff
@@ -86,3 +86,7 @@ func (e ErrorModel) Error() string {
 func (e ErrorModel) Cause() error {
 	return errors.New(e.Because)
 }
+
+func UnSupportedParameter(param string) {
```
Comment (vrothberg) [inline pkg/api/handlers/utils/errors.go]: Please add a comment.

**Comment by vrothberg** [comment]
The images job seems to fail on many PRs at the moment, so I guess that's unrelated. But we should investigate.

**Comment by TomSweeneyRedHat** [comment]
LGTM other than the other comments

**Comment by baude** [comment]
fixed up and ready for merge

**Pull request merged**